lib/api/websiteGet.js

```diff
@@ -9,6 +9,36 @@ import validateHeaders from '../utilities/validateHeaders';
 import { pushMetric } from '../utapi/utilities';
 
 /**
+ * _extractRedirectInfo - extract information from location for a redirect
+ * @param {string} url - location to redirect to
+ * @return {object} redirectInfo - select key/values stored in
+ * WebsiteConfiguration for a redirect -- protocol, replaceKeyWith and hostName
+ */
+function _extractRedirectInfo(url) {
+    const redirectInfo = {};
+    let location;
+    if (url.startsWith('https://')) {
+        redirectInfo.protocol = 'https';
+        location = url.slice(8);
+    } else if (url.startsWith('http://')) {
+        redirectInfo.protocol = 'http';
+        location = url.slice(7);
+    }
+    const firstSlash = location.indexOf('/');
+    const endIndex = firstSlash !== -1 ? firstSlash : location.length;
+    // anything before first slash will be host
+    redirectInfo.hostName = location.slice(0, endIndex);
+    // anything after first slash will be object key
```

Comment:
as discussed, this is not needed.  can send redirectKey as empty string to redirect request function and treat whole url as hostname.